specification.md

```diff
@@ -220,19 +221,20 @@ mixed Agents from different vendors.
 
 OpAMP supports the following functionality:
 
-* Remote configuration of the Agents.
-* Status reporting. The protocol allows the Agent to report the properties of
+- Remote configuration of the Agents.
```

Comment:
FYI, your editor changed `*` to `-` everywhere, which makes the diff larger than it should be.
```suggestion
* Remote configuration of the Agents.
```

Reply:
yeah :( i couldn't figure out how it decided what to change here... i can change it back if it really matters

Reply:
I don't have strong opinions, but I generally try to leave the file how I found it.